app/static/js/userManagement.js

```diff
@@ -1,22 +1,16 @@
 import searchUser from './searchUser.js'
-function callback(selected) {
-  $("#searchAdmin").submit();
+function callbackAdmin(selected) {
+    submitRequest("addCeltsAdmin", selected.username)
+}
+function callbackStudentStaff(selected) {
+    submitRequest("addCeltsStudentStaff", selected.username)
 }
 
 $(document).ready(function() {
   // Admin Management
-  var searchElements = [
-    // Search Input ID               Button ID                 Category
-    ['searchCeltsAdminInput',       'addCeltsAdmin',          'instructor'],
-    ['searchCeltsStudentStaffInput','addCeltsStudentStaff',   'student'],
-    ['removeCeltsAdminInput',       'removeCeltsAdmin',       'admin'],
-    ['removeCeltsStudentStaffInput','removeCeltsStudentStaff','studentstaff']
-  ];
-  $.each(searchElements, function(i,arr) {
-      let [inputId, btnId, category] = arr
-      $("#"+inputId).on("input", () => searchUser(inputId, callback, false, null, category))
-      $("#"+btnId).on("click", () => submitRequest(btnId, $("#"+inputId).val()))
-  });
+  $("#searchCeltsAdminInput").on("input", () => searchUser("searchCeltsAdminInput", callbackAdmin, false, null, 'instructor'))
+  $("#searchCeltsStudentStaffInput").on("input", () => searchUser("searchCeltsStudentStaffInput", callbackStudentStaff, false, null, "student"))
```

Comment:
Let's reformat these to match the other event handlers, now that they are the same as the others

Reply:
resolved